### ISSUE TITLE
Grid tools structure not ideal for a11y

### ISSUE THREAD

**Issue by oysteinmoseng** [highcharts/sonification-studio#18]
The add rows fieldset could arguably be a violation of WCAG 2.5.3 Label in Name with the current structure, as the spinbox does not have a visible label other than the fieldset legend, and the fieldset legend does not match the spinbox accessible name.

Also the fill column controls do not have the same semantic setup, and no fieldset legend, which could be confusing.

The best way to fix this is probably with a disclosure/popup style design so we can add a proper fieldset legend and labels to everything.

This can perhaps wait for the grid replacement, since the add rows functionality will be included in the grid itself.
